green/output.py

```diff
@@ -167,6 +171,9 @@ def writeln(self, text=''):
     def write(self, text):
         if type(text) == bytes:
             text = text.decode('utf-8')
+        # Compensate for windows' anti-social unicode behavior
+        if self._ascii_only_output:
+            text = unidecode(text)
```

Comment:
Sorry for the delay.  I have been just absolutely swamped.

I suggest just changing line 176 to read:

    text = unidecode(unicode(text))

That may be work and we could be done with this and merge it.